src/bp-activity/bp-activity-filters.php

```diff
@@ -2283,9 +2292,9 @@ function bp_activity_edit_update_document( $document_ids ) {
  * @param int   $activity_id Activity ID.
  *
  * @return mixed
- * @since BuddyBoss 1.5.0
+ * @since BuddyBoss 1.7.1
  */
-function bp_nouveau_remove_edit_activity_entry_buttons( $buttons, $activity_id ) {
```

Comment:
@asaquzzamanbb Add this function into the deprecation file.

Reply:
@chetansatasiya 

@janakkaneriya007 is the responsible of this function. By mistake i have change the function prefix. just revert the function prefix here
https://github.com/buddyboss/buddyboss-platform/pull/2290/commits/e2d82c2042a004cedaf97b3164e161ee4472f3b7